understanding/21/pointer-gestures.html

```diff
@@ -20,14 +20,16 @@ <h2>Intent of this Success Criterion</h2>
 			</figure>
 
 			<p>Examples of path-based gestures include swiping, sliders and carousels dependent on the direction of interaction, and other gestures which trace a prescribed path such as drawing a specific shape. Such paths may be drawn with a finger or stylus on a touchscreen, graphics tablet, or trackpad, or with a mouse, joystick, or similar pointer device.</p>
+     
+      <h3>Disambiguation of Pointer Gestures and Dragging</h3>
```

Comment:
A simpler heading (think COGA) would be "The difference between pointer gestures and dragging"